OpenRA.Mods.Common/Traits/RejectsOrders.cs

```diff
@@ -10,21 +10,26 @@
 #endregion
 
 using System.Collections.Generic;
+using System.Linq;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Can be used to make a unit partly uncontrollable by the player.")]
 	public class RejectsOrdersInfo : ConditionalTraitInfo
 	{
-		[Desc("Possible values include Attack, AttackMove, Guard, Move.")]
+		[Desc("Explicit list of rejected orders. Leave empty to reject all minus those listed under Except.")]
+		public readonly HashSet<string> Reject = new HashSet<string>();
+
+		[Desc("List of orders that should *not* be rejected.")]
```

Comment:
"Also overrides other instances of this trait's Reject fields."

I feel that fact worthy to be documented. With that, I gladly  :+1: this. (The solution itself is top-notch, but this aspect might end up overlooked at first.)